### PR TITLE
Added mappedbus

### PR DESCRIPTION
**Pull request by MikaelJ1**

### CODE REVIEW

File: projects.yml [hunk 1]
```diff
@@ -431,6 +431,19 @@ popular:
...
+    url: http://mappedbus.io/
+    tags:
+      - java
+      - high-throughput
+      - low-latency
+      - message-passing
+      - ipc
+    links:
+      - text: "Source on GitHub"
+      - url: http://github.com/caplogic/mappedbus
```
Comment (strzalek) [inline projects.yml]: Overall looks good, but I have an issue with links section. The idea was to collect links that are not the "go-to", easy to google ones like homepage urls.

Do you have a link to a some kind of presentation or blog post describing this lib from a 3rd party perspective maybe?

Comment (MikaelJ1) [inline projects.yml]: Unfortunately don't have any such link - how about this one where a few people ask some questions about the library?: https://www.reddit.com/r/java/comments/3624hj/mappedbus_ipc_between_multiple_java_processes/

Comment (strzalek) [inline projects.yml]: Nah. Okay let's make an exception to the rule (I will have soooo much
trouble because of that ;) ).

**Comment by strzalek** [comment]
Thanks!

File: projects.yml (hunk 1 above)
Comment (caplogic) [inline projects.yml]: Great! :)

**Pull request merged**